automation/src/test/java/com/microsoft/identity/common/test/automation/utility/TokenRequest.java

```diff
@@ -161,4 +163,12 @@ public Boolean getForceRefresh() {
     public void setForceRefresh(Boolean forceRefresh) {
         this.mForceRefresh = forceRefresh;
     }
+
+    public void setFamilyClientId(String familyClientId){
```

Comment:
I would suggest you to add 'lombok' as dependency to avoid these setter/getter.

Reply:
I'm not aware of this library. In the interest of time for this release, I would pass on this and may be can take a look into the library in future